main.py

```diff
@@ -40,8 +40,14 @@ def main():
     config = utils.init_result_folder(config_filepath, config)
     experiment_path = config["experiment_path"]
 
-    scenario_params_list = config["scenario_params_list"]
+    scenario_params_list = utils.get_scenario_params_list(
+        config["scenario_params_list"])
     n_repeats = config["n_repeats"]
+    
+    print('\nScenario(s) to process:', )
+    for ind, scenar in enumerate(scenario_params_list):
```

Comment:
I would use `scenario_id`, `scenario_params `to be consistent with the code below.

Reply:
Done